pkg/container-disk/container-disk.go

```diff
@@ -187,22 +187,18 @@ func generateContainersHelper(vmi *v1.VirtualMachineInstance, podVolumeName stri
 			diskContainerName := fmt.Sprintf("volume%s", volume.Name)
 			diskContainerImage := volume.ContainerDisk.Image
 			resources := kubev1.ResourceRequirements{}
+			resources.Limits = make(kubev1.ResourceList)
+			resources.Requests = make(kubev1.ResourceList)
+			resources.Limits[kubev1.ResourceMemory] = resource.MustParse("40M")
+			resources.Requests[kubev1.ResourceCPU] = resource.MustParse("10M")
+			resources.Requests[kubev1.ResourceEphemeralStorage] = resource.MustParse("50M")
```

Comment:
Should we add resource limits for the ephemeral-storage as well?

Reply:
That's a good question :)
Should we?